mava/wrappers/gym.py

```diff
@@ -0,0 +1,312 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import warnings
+from typing import Any, Callable, Dict, Optional, Tuple
+
+import gym
+import numpy as np
+from gym import spaces
+from gym.vector.utils import write_to_shared_memory
+from numpy.typing import NDArray
+
+# Filter out the warnings
+warnings.filterwarnings("ignore", module="gym.utils.passive_env_checker")
+
+
+class GymRwareWrapper(gym.Wrapper):
+    """Wrapper for rware gym environments."""
+
+    def __init__(
+        self,
+        env: gym.Env,
+        use_shared_rewards: bool = False,
```

Comment:
Since we mostly focus on cooperative MARL, I think we can just have `use_shared_rewards` as True by default.

```suggestion
        use_shared_rewards: bool = True,
```